web_frontend/osmaxx/excerptexport/models/bounding_geometry.py

```diff
@@ -47,9 +46,9 @@ def __str__(self):
 
 class BBoxBoundingGeometry(BoundingGeometry):
     def __init__(self, *args, **kwargs):
-        attribute_names = ['north', 'east', 'south', 'west']
-        if are_all_keys_in(kwargs, attribute_names) \
-                and not are_all_keys_in(kwargs, ['south_west', 'north_east']):
+        attribute_names = {'north', 'east', 'south', 'west'}
+        if set(attribute_names).issubset(kwargs) \
+                and not {'south_west', 'north_east'}.issubset(kwargs):
```

Comment:
I like the tests, no argument from me there. For me, just reading through the code, `are_all_keys_in` with `arg1`=my dictionary and `arg2`=my list of keys to test against makes it faster to understand what goes on than when using the inline version. I'm also not arguing against the implementation itself, I like that much better, but I would keep the encapsulation in a separate function.